xds/internal/balancer/xds.go

```diff
@@ -33,16 +33,14 @@ import (
 	"google.golang.org/grpc/grpclog"
 	"google.golang.org/grpc/resolver"
 	"google.golang.org/grpc/serviceconfig"
+	xdsinternal "google.golang.org/grpc/xds/internal"
 	"google.golang.org/grpc/xds/internal/balancer/edsbalancer"
 	"google.golang.org/grpc/xds/internal/balancer/lrs"
 	cdspb "google.golang.org/grpc/xds/internal/proto/envoy/api/v2/cds"
 	edspb "google.golang.org/grpc/xds/internal/proto/envoy/api/v2/eds"
 )
 
-const (
-	defaultTimeout = 10 * time.Second
-	xdsName        = "xds_experimental"
```

Comment:
Actually let's keep the names separate....

Underscore is not a valid character for URLs. So the resolver will use `xds-experimental`, and balancer will keep using `xds_experimental`..

Reply:
Done. PTAL.

Reply:
Balancer name should remain "xds_experimental"

Reply:
My bad. Fixed it now.